plugin-dev/Source/Sentry/Private/Android/Jni/SentryJniAndroid.cpp

```diff
@@ -38,7 +39,14 @@ JNI_METHOD jobject Java_io_sentry_unreal_SentryBridgeJava_onBeforeSend(JNIEnv* e
 	USentryEvent* EventToProcess = USentryEvent::Create(MakeShareable(new SentryEventAndroid(event)));
 	USentryHint* HintToProcess = USentryHint::Create(MakeShareable(new SentryHintAndroid(hint)));
 
-	return handler->HandleBeforeSend(EventToProcess, HintToProcess) ? event : nullptr;
+	USentryEvent* ProcessedEvent = EventToProcess;
+	if (!FUObjectThreadContext::Get().IsRoutingPostLoad)
```

Comment:
Can't we skip the whole method in that case?

Reply:
Nitpick but I'd probably invert the check, bail early and print some debug logs. So users have a way to follow why their beforeSend is not getting called.

Reply:
Fair enough, the check is now inverted and prints a log message explaining why custom `beforeSend` handling was skipped.